### PR TITLE
making utilities functions available

### PR DESCRIPTION
**Pull request by scottnath**
Opening up the utilities file to use those functions in other modules.

@sergesemashko could you check this for me please?

### CODE REVIEW

**Comment by scottnath** [comment]
@beynya can you check this out please?

**Comment by beynya** [comment]
@scottnath not now, sorry :(

File: index.js [hunk 1]
```diff
@@ -44,4 +44,9 @@ function patternImporter (options) {
 
 }
 
-module.exports = patternImporter;
+module.exports = exports = patternImporter;
```
Comment (sergesemashko) [inline index.js]: as far as I know `exports` is alias of  `module.exports`, `module.exports = patternImporter` should be enough.
Also, I think it makes sense to export utils clearly as a property of object:

``` js
module.exports = {
  patternImporter: patternImporter,
  utils: utils
}
```

or I saw some modules do even like this:

``` js
module.exports = function () {
  return function (export) {
    if (export === 'utils') {
      // can be received by require('patternImporter')('utils')
      return require('./lib/utils');
    }
    if (export === 'streamHandler') {
      // can be received by require('patternImporter')('streamHandler')
      return patternImporter;
    }
  } 
}
```

Comment (scottnath) [inline index.js]: I think I want to switch to this:

```
module.exports = {
  patternImporter: patternImporter,
  utils: utils
}
```

for clarity and to allow us to open up other functions to use via NPM

File: index.js [hunk 2]
```diff
@@ -44,4 +44,10 @@ function patternImporter (options) {
 
 }
 
-module.exports = patternImporter;
+module.exports = {
```
Comment (scottnath) [inline index.js:47]: @sergesemashko my change

**Comment by sergesemashko** [comment]
:+1:

**Pull request merged**